projects/plugins/mu-wpcom-plugin/changelog/update-launchpad-add_new_page-href

```diff
@@ -0,0 +1,4 @@
+Significance: minor
```

Comment:
Are these two changelogs needed? There don't seem to be any changes here right?

Reply:
Mmm I don't know, the commit hook said the packages were affected 😒 Should I remove them?